cmd/tsbs_generate_queries/main.go

```diff
@@ -79,10 +79,11 @@ func getGenerator(format string, start, end time.Time, scale int) utils.DevopsGe
 		return mongo.NewDevops(start, end, scale)
 	} else if format == "mongo-naive" {
 		return mongo.NewNaiveDevops(start, end, scale)
-	} else if format == "timescaledb" {
+	} else if format == "timescaledb" || format == "postgresql" {
```

Comment:
My only worry here is that now there is no equivalent in `generate_data` which people might find confusing.